ride-cleansing/src/main/java/org/apache/flink/training/exercises/ridecleansing/RideCleansingExercise.java

```diff
@@ -80,7 +81,8 @@ public JobExecutionResult execute() throws Exception {
     public static class NYCFilter implements FilterFunction<TaxiRide> {
         @Override
         public boolean filter(TaxiRide taxiRide) throws Exception {
-            throw new MissingSolutionException();
+            return (GeoUtils.isInNYC(taxiRide.startLon, taxiRide.startLat)
```

Comment:
I thought I approved that in previous PR, not sure why I should review it second time.
You just need to apply first PR
https://github.com/fhtrancoso/flink-training/pull/1
and then build next lab on top of it.

Reply:
Because I did not merge the first PR. I did it now. And will do in this one.